docs/glossary.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Glossary
+---
+
+## Terms
+
+### button
```

Comment:
Let's remove 'button' and the link to it.  I think we can assume people know what a button is by now.